components/checkbox/Checkbox.js

```diff
@@ -22,6 +22,7 @@ const factory = (Check) => {
       onMouseEnter: PropTypes.func,
       onMouseLeave: PropTypes.func,
       style: PropTypes.object,
+      testid: PropTypes.string,
```

Comment:
Is there no other way to do it? We want to remove all forks, unnecessary overhead of maintaining these repos.

Reply:
Currently we have 2 checkboxes - v1 (this one) and v2 (a custom one). I first tried to use the second one and pass the testid to it, but it will require more effort as its design is different and props needs to be modified as well. I think it will be better if we take up the dependency cleanup as a separate story.

Reply:
Closing this PR. As discussed, I'll add a wrapper div in dashboard component.